subprojects/build-cache/build.gradle.kts

```diff
@@ -7,6 +7,7 @@ dependencies {
     api(project(":build-cache-base"))
     api(project(":snapshots"))
 
+    implementation(project(":build-cache-packaging"))
```

Comment:
Why do we need this now? Can we put stuff in `:build-cache-base` instead?

Reply:
We need dependency to a `BuildCacheEntryPacker`. But if I move the interface to a `:build-cache-base` then I have to add `:snapshots` dependency to a `:build-cache-base`, since `FileSystemSnapshot` is referenced from `BuildCacheEntryPacker.pack` method.

What is the lesser evil here?